test/integration/sanity/clear-vars-sandbox.test.js

```diff
@@ -83,24 +71,21 @@ describe('Clear vars sandbox', function () {
         expect(testrun).be.ok();
         expect(testrun.test.calledThrice).be.ok();
 
-        expect(testrun.test.getCall(0).args[0]).to.be(null);
-        expect(_.get(testrun.test.getCall(0).args[2], '0.result.globals.tests')).to.eql({
-            'Variable substitution from env': true,
-            'Variable substitution from global': true
-        });
+        var first = testrun.test.getCall(0),
```

Comment:
make these separate it blocks!! how will we know post failure what failed? and make the `it` description clean enough to know what was tested